packages/core/admin/admin/src/pages/SettingsPage/pages/Roles/EditPage/components/ConditionsButton/index.js

```diff
@@ -55,4 +55,4 @@ ConditionsButton.propTypes = {
 // This is a styled component advanced usage :
 // Used to make a ref to a non styled component.
 // https://styled-components.com/docs/advanced#caveat
-export default styled(ConditionsButton)``;
+export default styled(ConditionsButton);
```

Comment:
Could you explain the reason for this change? The styled components docs do include the empty string. Removing it changes the TS type of the export, so I suspect it can have some importance.